add roles to authorization (#392)
GemeenteUtrecht/zaakafhandelcomponent: backend/doc/authorization.rst

* document
* report

Each object type has its required shape for a blueprint permission.

> It is explained in the details in the "Blueprint permission" subsection.

Reviewer: You can label that title in Sphinx and create a clickable link to it.

E.g. use this label setup:

```rst
.. _authorization_blueprints:

Blueprint permissions
---------------------
```

then you can do:

```suggestion
It is explained in the details in :ref:`authorization_blueprints`.
```

      python src/manage.py add_blueprint_permissions_for_zaaktypen

Roles
^^^^^

Blueprint permissions link the shape of the object (blueprint) with the set of operations.

> Sets of operations represent roles. It is possible to include several (or all) operations in one role.

Reviewer: ```suggestion
Roles represent sets of operations. It is possible to include multiple (or all) operations in one role.
```

> Using roles makes it easy to add the same permissions for different blueprints.

Reviewer: ```suggestion
Using roles simplifies re-using the same permissions for different blueprints.
```
